include/godot_cpp/variant/vector3.hpp

```diff
@@ -403,15 +403,15 @@ Vector3 Vector3::normalized() const {
 
 bool Vector3::is_normalized() const {
 	// use length_squared() instead of length() to avoid sqrt(), makes it more stringent.
-	return Math::is_equal_approx(length_squared(), 1.0, UNIT_EPSILON);
+	return Math::is_equal_approx(length_squared(), 1.0f, (real_t)UNIT_EPSILON);
```

Comment:
While `1.0f` can be implicitly and perfectly casted to double, this function has both float and double overrides, so we really should be explicit here:

```suggestion
	return Math::is_equal_approx(length_squared(), (real_t)1.0, (real_t)UNIT_EPSILON);
```

Reply:
I might change everything to be explicit then, seems like the safer way forward.